src/pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -105,7 +114,7 @@ function registerCanonicalToken(
 	* @param destinationChain the name of the chain to deploy the TokenManager and standardized token to.
 	* @param gasValue the amount of native tokens to be used to pay for gas for the remote deployment.
 	* At least the amount specified needs to be passed to the call
-	* @dev `gasValue` exists because this function can be part of a multicall involving multiple functions that could make remote contract calls.
+	* @dev `gasValue` exists because this function can be part of a multi call involving multiple functions that could make remote contract calls.
```

Comment:
I think the technical term is `multicall`, but I'm not sure, can you triple check? https://github.com/mds1/multicall